File: mainActivity/src/main/java/com/sierrawireless/avphone/task/SyncWithAvParams.java
package com.sierrawireless.avphone.task;

import com.sierrawireless.avphone.MainActivity;
import com.sierrawireless.avphone.model.CustomDataLabels;

public class SyncWithAvParams {
    public String deviceId;
    public String imei;
    public String mqttPassword;
    public String iccid;
    public String deviceName;
    public CustomDataLabels customData;
    public MainActivity activity;
    public boolean delete = false;
}

File: mainActivity/src/main/java/com/sierrawireless/avphone/task/SyncWithAvParams.kt
package com.sierrawireless.avphone.task

import com.sierrawireless.avphone.MainActivity
import com.sierrawireless.avphone.model.CustomDataLabels

class SyncWithAvParams {
    var deviceId: String? = null
    var imei: String? = null
    var mqttPassword: String? = null
    var iccid: String? = null
    var deviceName: String? = null
    var customData: CustomDataLabels? = null
    var activity: MainActivity? = null
}
